lib/zendesk_apps_support/validations/screenshots.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'image_size'
+
+module ZendeskAppsSupport
+  module Validations
+    module Screenshots
+      MAX_FILES = 3
+      SCREENSHOT_WIDTH = 1024
+      SCREENSHOT_HEIGHT = 768
+
+      class << self
+        def call(package)
+          errors = []
+          screenshot_paths = Dir["#{package.path_to('assets')}/*"].grep(/screenshot-\d.png$/)
+          file_count = screenshot_paths.count
+          if file_count > MAX_FILES
+            errors << ValidationError.new('screenshots.excessive_files',
+                                          file_count: file_count,
+                                          supported_count: MAX_FILES)
```

Comment:
This is not a blocker. Personally I would prefer to [align the parameters of a method call](https://github.com/rubocop-hq/ruby-style-guide#no-double-indent) if they span more than one line.

Reply:
I'm not sure I understand this comment. Isn't your suggestion what I'm already doing?

Reply:
Definitely. I was referring to the # good (normal indent) where we go

```
errors <<
something.method(
  key_1: '...',
  key_2: '...'
)
```

But yours look good too! :)